### ISSUE TITLE
Can I load vanilla js via API?

### ISSUE THREAD

**Issue by iliakan** [CompuIves/codesandbox-client#1083]
Or only react template?

**Comment by CompuIves** [comment]
You mean specifying a template for the Define API? You can specify vanilla template by adding `parcel-bundler` to the devDependencies.

We should have documentation on this. I will write that today!

**Comment by iliakan** [comment]
Hi @CompuIves,

I'm webmaster of https://javascript.info, the Modern Javascript Tutorial. 
Right now we use plnkr.co for all examples, but I'm thinking to move to codesandbox.

But I need something like "barebone" JS/HTML. Without any templates and, preferably, without `package.json` also. So that people don't get distracted by what they don't need.

Can we organize something like that?

**Comment by CompuIves** [comment]
Yup! I've seen some requests for a template that's just static, no transpiling and bundling going on. Does that make sense for your use case?

**Comment by iliakan** [comment]
Yes, to learn how to code, people need to see "templateless" stuff first.

"As-is" `index.html` with script tags (`<html>...<script src="my.js">...</html>`), and `my.js` in the same folder, in exactly the same form they have on their disk.

I guess not only me, but any HTML or CSS or JS tutorial would want exactly that. 
Bundling and other "under the hood magic" comes later, when they are familiar with HTML/CSS/JS.

**Comment by CompuIves** [comment]
That makes sense and isn't too hard to add these days I think. A `package.json` would still be required though, because we infer sandbox data like title from it.

**Comment by iliakan** [comment]
Allow it to be an empty object or `null`? Then you can infer "zero data" from it, just what a barebone example requires.

**Comment by iliakan** [comment]
@CompuIves can you please tell me, are you going to implement that in the nearest future, like a few days?

Totally no means to rush you, I'm refactoring the tutorial engine and would like to know whether to wait and port it to your sandbox or put efforts into staying with next.plnkr.co for now.

Thank you ;)

**Comment by craigcartmell** [comment]
Hey @CompuIves - Did you ever get around to writing any docs on this?

When you say we just need to add parcel-bundler to devDependencies, do you mean within the package.json we send in the define API request?

**Comment by craigcartmell** [comment]
Side question, are all of the templates [here](https://codesandbox.io/s/) stored on github?

I can see for example Gatsby is, but is the Vanilla one for example?

**Comment by lbogdan** [comment]
Hey @iliakan ,

Did the [static template](https://codesandbox.io/s/github/codesandbox-app/static-template) solve your problem? If not, do you still need this?

**Comment by iliakan** [comment]
@lbogdan Maybe it solves. I'm not looking into it now.
There's another problem in the UI. It's overloaded, especially to the left.

A bunch of stuff should be removed/collapsed or made removable. It's crazy - I open the editor and the file list just doesn't fit. Less important stuff does.
https://jmp.sh/vLtHp2T

**Comment by iliakan** [comment]
As far as I can see, static template still includes an "extra" codesandbox JS, right?

**Comment by SaraVieira** [comment]
Hey

The static template has no bundling or anything, just one JS file to keep the preview in sync with the editor, you can remove this by deploying to netlifly

In terms of build, it does not have anything so it should fix this issue

We also did an overhaul in the UI and removed ads
